pull_request_template.md

```diff
@@ -1,8 +1,16 @@
+## Ticket
+Please include a link to the ticket
+
 ## Description
 Please include a summary of the changes
 
-## Ticket
-Please include a link to the ticket
```

Comment:
I tend to think that what the developer has to say will be more informative than the ticket, which I consider just a useful reference. So personally, I'd lean towards keeping the description up top, but I welcome different views.

Reply:
Hmm, my thought here was that we link the ticket first, and then have all the relevant information grouped after it so it won't break the reviewer's "flow" quite as much.

Reply:
Hm, interesting! I'm thinking about it from the opposite end: that the checkboxes are intended as a last-minute check for the PR's creator, to confirm they've done everything that needs to be done.

Reply:
I don't have a strong opinion about it so I'm happy to reverse it. 😄